# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\scipy\optimize\_direct_py.py
# hypothesis_version: 6.140.2

[1e-16, 1e-06, 0.0001, -100, 1000, 'Forced stop', 'Invalid arguments', 'Out of memory', 'direct', 'maxfun is too large', 'maxfun must be > 0.', 'maxiter must be > 0.']